frontend/packages/webterminal-plugin/src/cloud-shell/WebTerminalConfiguration.tsx

```diff
@@ -0,0 +1,369 @@
+import * as React from 'react';
```

Comment:
All other packages uses src/components/...

If not can we maybe move this here into `webterminal-plugin/src/configuration/WebTerminalConfiguration.tsx` so that is has a different folder then the terminal UI itself?